deploy/nuget/devel-reduced/prepare_dal_nuget.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+create_package() {
+    # Args:
+    # 1 - system
+    if [ $1 = "lnx" ]; then
+        platform=linux-x64
```

Comment:
I guess in nuget they use '<os>-x64' intentionally, instead of how conda uses '<os>-64'. Just an observation :)